modules/statics/test/unit/coins.ts

```diff
@@ -10,6 +10,16 @@ describe('CoinMap', function () {
   it('should have iterator', function () {
     [...coins].length.should.be.greaterThan(100);
   });
+
+  it('should report if it contains coin', () => {
```

Comment:
What do these tests have to do with the change being made?

Reply:
method `has` was added to CoinMap to check for the existence of some name in the map.